.devcontainer/devcontainer.json

```diff
@@ -1,29 +1,24 @@
 // For format details, see https://aka.ms/devcontainer.json. For config options, see the
 // README at: https://github.com/devcontainers/templates/tree/main/src/python
 {
-	"name": "Python 3",
-	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
-	"image": "mcr.microsoft.com/devcontainers/python:1-3.12-bullseye",
-	"runArgs": ["--add-host=host.docker.internal:host-gateway"],
-	"features": {
+  "name": "Journal API",
+  "dockerComposeFile": ["./docker-compose.yml"],
+  "features": {
 		// TODO: Add one of these cloud CLI tools based on your needs:
         // "ghcr.io/devcontainers/features/azure-cli:1": {},
         // "ghcr.io/devcontainers/features/aws-cli:1": {},
         // "ghcr.io/devcontainers/features/gcloud:1": {}
-	}
+	},
+  
+  "service": "devcontainer",
+  "workspaceFolder": "/workspaces/"
```

Comment:
You need to add "runServices": ["postgres"], here to tell VS Code to also start the postgres service